### PR TITLE
fix(TBD-12471):[BUG] EMR62: Runtime exception on Spark Batch job with…

### PR DESCRIPTION
**Pull request by cbadillo1603**
… tS3Configuration component

**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.emr62x/plugin.xml [hunk 1]
```diff
@@ -438,6 +438,10 @@
 		<libraryNeeded context="plugin:org.talend.hadoop.distribution.emr62x"
             id="aws-java-sdk-core-1.11.880.jar" name="aws-java-sdk-core-1.11.880.jar"
             mvn_uri="mvn:com.amazonaws/aws-java-sdk-core/1.11.880"/>
+
```
Comment (hliashchuk) [inline main/plugins/org.talend.hadoop.distribution.emr62x/plugin.xml:441]: This is also missing on maintenance

File: main/plugins/org.talend.hadoop.distribution.emr62x/src/main/java/org/talend/hadoop/distribution/emr62x/EMR62xDistribution.java [hunk 2]
```diff
@@ -370,7 +370,7 @@ public boolean doSupportExtendedAssumeRole() {
     
     @Override
     public boolean doSupportEMRFS() {
-        return true;
+        return false;
```
Comment (sponomarova) [inline main/plugins/org.talend.hadoop.distribution.emr62x/src/main/java/org/talend/hadoop/distribution/emr62x/EMR62xDistribution.java:373]: @cbadillo1603 , could you please explain why EMRFS was disabled?

**Review by firefryy — APPROVED** [review]
lgtm

File: main/plugins/org.talend.hadoop.distribution.emr62x/src/main/java/org/talend/hadoop/distribution/emr62x/EMR62xDistribution.java (hunk 2 above)
Comment (cbadillo1603) [reply main/plugins/org.talend.hadoop.distribution.emr62x/src/main/java/org/talend/hadoop/distribution/emr62x/EMR62xDistribution.java:373]: It is disabled as a hotfix for the Tech preview. We will handle it better in the future GA

**Pull request merged**